### DISCUSSION TITLE
Can we incorporate Register and save user to database?

### DISCUSSION THREAD

**Discussion by yukirivero** [ditdot-dev/vue-flow-form#227] [Q&A] [answered]
Can we incorporate Register and save user to database?

**Comment by spinn** [comment, marked as answer]
Hi @yukirivero,

this is out of scope for Vue Flow Form itself, but yes, you could send the collected data to your backend of choice in the `complete` event and save everything to the database. You can see an example of how to send data to your backend here:

https://github.com/ditdot-dev/vue-flow-form/blob/master/examples/questionnaire/Example.vue#L290-L310
